Make Saline accessible by /saline with Apache in Uyuni/MLM

cbosdo opened the pull request:
Apache now proxies saline's 8216 port on /saline. Change the formula and dashboards to match this.
The commits changing this are:
* https://github.com/uyuni-project/uyuni-tools/pull/482/commits/b091555312039c12f26a9ed87b0b0cce9b01fed9
* https://github.com/uyuni-project/uyuni/pull/9827/commits/ad04eeb9bf80799b3e4e52248a437c62830cc727

vzhestkov commented on formulas/states/saline-prometheus/init.sls:
```diff
@@ -4,7 +4,7 @@ saline-prometheus-cfg:
   - contents: |
       - targets:
-        - {{ salt['pillar.get']('mgr_server') }}:{{ salt['pillar.get']('prometheus:saline_port') }}
+        - {{ salt['pillar.get']('mgr_server') }}/saline }}
 {%- if salt['pillar.get']('prometheus:saline_https_connection', False) %}
         labels:
           __scheme__: https
```
For some reason I can't make working the config from your PR, maybe I missed something important, but the suggested below works fine for me:
```suggestion
        - {{ salt['pillar.get']('mgr_server') }} }}
        labels:
          __metrics_path__: /saline/metrics
{%- if salt['pillar.get']('prometheus:saline_https_connection', False) %}
          __scheme__: https
```

cbosdo replied:
I've seen that in my tests and your suggestion is better than my solution

vzhestkov commented on formulas/states/saline-grafana/files/mgr-saline-state-jobs.json.jinja:
```diff
@@ -406,7 +406,7 @@
           "exemplar": false,
-          "expr": "max by (fun,test,mods) (salt_state_jobs{instance=\"$server:$saline_port\",status=\"targeted\"})",
+          "expr": "max by (fun,test,mods) (salt_state_jobs{instance=\"$server/saline\",status=\"targeted\"})",
           "format": "table",
```
And such changes also doesn't work in my case, but it works fine if left `$saline_port` with the queries and just change it in constant to `80`

cbosdo replied:
since it's using https it'll be 443, I suppose.

vzhestkov reviewed (changes requested):
Left couple of comments in line and I also not sure about the title saying explicitly port 80, why we are not switching to `443` to avoid exposing plain traffic between the server and monitoring host?

cbosdo committed:
Replace 8216 port with the /saline path on 443 port

Apache now proxies saline's 8216 port on /saline. Change the formula and
dashboards to match this.

cbosdo commented:
Doc fix PR: https://github.com/uyuni-project/uyuni-docs/pull/3862

vzhestkov reviewed (approved):
LGTM. Thanks a lot for taking care of it :+1:
